app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  def update_user_post_counter
+    author.increment(:postscounter)
+  end
```

Comment:
- [ ] [OPTIONAL] Great job implementing the `update_user_post_counter` method. A way to make it even cooler would be to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the #update_user_post_counter` method every time a new post is created and saved to the database, then make the update_user_post_counter method private
### HINT 👇 
```ruby
after_save :update_post_counter

  private

  def update_user_post_counter
    author.increment!(:postscounter)
  end
```